chef/cookbooks/crowbar-pacemaker/recipes/drbd.rb

```diff
@@ -26,7 +26,10 @@
 
 if claimed_disks.empty? and not unclaimed_disks.empty?
   unclaimed_disks.each do |disk|
-    if disk.claim(claim_string)
+    %x{lsblk #{disk.name} --noheadings --output MOUNTPOINT | grep -q -v ^$}
```

Comment:
Isn't that some thing that should be done in `BarclampLibrary::Barclamp::Inventory::Disk.unclaimed()` ? That way every consumer who needs an unclaimed disk has this fix.